src/simulation/MorphoAaveV3Simulator.ts

```diff
@@ -597,9 +598,15 @@ export class MorphoAaveV3Simulator extends MorphoAaveV3DataEmitter {
     const morphoSupplyInP2P = marketData.morphoSupplyInP2P.add(p2pAmount); // Matched
     const poolBorrow = marketData.poolBorrow.add(poolAmount);
     const totalMorphoBorrow = marketData.totalMorphoBorrow.add(amount);
-    const poolLiquidity = marketData.poolLiquidity.sub(p2pAmount); // Matched
+    const poolLiquidity = amount.eq(marketData.poolLiquidity)
+      ? constants.Zero
+      : marketData.poolLiquidity.sub(poolAmount);
```

Comment:
```suggestion
    const poolLiquidity = amount.eq(marketData.poolLiquidity)
      ? constants.Zero
      : marketData.poolLiquidity.sub(amount);
```
all the underlying are coming / go to the pool